src/headers.js

```diff
@@ -30,21 +30,20 @@ export class Headers {
    * This method parses that string into a user-friendly key/value pair object.
    */
   static parse(headerStr : string) : Headers {
-    var headers = new Headers();
+    let headers = new Headers();
     if (!headerStr) {
       return headers;
     }
 
-    var headerPairs = headerStr.split('\u000d\u000a');
-    for (var i = 0; i < headerPairs.length; i++) {
-      var headerPair = headerPairs[i];
+    let headerPairs = headerStr.split('\u000d\u000a');
+    for (let headerPair of headerPairs) {
```

Comment:
I certainly agree that for..of is nicer here, but we've intentionally avoided it because babel generates a lot of code to handle iterators, which bloats the lib.